src/patch.js

```diff
@@ -1,91 +1,67 @@
+      console.info('SFDS form created!')
+      form.element.classList.add(wrapperClass, 'd-flex', 'flex-column-reverse', 'mb-4')
 
-  AddressComponent.schema = (...extend) => {
-    // Get the "base" schema
-    const schema = defaultSchema(...extend)
+      patchScrollOnPageNav(form)
 
-    // Overwrite the components array
-    //
-    // Note: if you provide `components: [...]` in the call to defaultSchema()
-    // above, the merge operation keeps some of the removed items, like the
-    // country input, intact.
-    schema.components = [
-      {
-        label: 'Address line 1',
-        tableView: false,
-        key: 'address1',
-        type: 'textfield',
-        input: true
-      },
-      {
-        label: 'Address line 2',
-        tableView: false,
-        key: 'address2',
-        type: 'textfield',
-        input: true
-      },
-      {
-        label: 'City',
-        tableView: false,
-        key: 'city',
-        type: 'textfield',
-        input: true
-      },
-      {
-        type: 'columns',
-        columns: [
-          {
-            width: 4,
-            components: [
-              {
-                label: 'State',
-                tableView: false,
-                key: 'state',
-                type: 'textfield',
-                input: true
-              }
-            ]
-          },
-          {
-            width: 2,
-            components: [
-              {
-                label: 'ZIP code',
-                tableView: false,
-                key: 'zip',
-                type: 'textfield',
-                input: true
-              }
-            ]
-          }
-        ]
-      }
-    ]
+      const model = { ...form.form }
+      patchAddressManualMode(model)
+      patchSelectMode(model)
+      form.form = model
 
-    return schema
+      return form
+    })
+  })
+}
+
+function patchAddressManualMode (model) {
+  const addresses = util.searchComponents(model.components, { type: 'address' })
+  for (const component of addresses) {
+    // FIXME no combination of these seems to make the nested
+    // fields render...
+    component.mode = 'manual'
+    component.enableManualMode = true
+    component.manualMode = true
   }
+}
 
-  // Lastly, redefine the getters for manualMode and manualModeEnabled so that
-  // the existing address component implementation only shows our inputs (and
-  // not the autocomplete UI)
-  Object.defineProperties(AddressComponent.prototype, {
-    manualMode: {
-      get () { return true }
-    },
-    manualModeEnabled: {
-      get () { return true }
-    }
-  })
+function patchSelectMode (model) {
+  const selects = util.searchComponents(model.components, { type: 'select' })
+  for (const component of selects) {
+    component.widget = 'html5'
+  }
+}
+
+function patchScrollOnPageNav (form) {
+  const scrollToForm = () => {
+    scrollIntoView(form.element, { behavior: 'smooth' })
+  }
+  form.on('nextPage', scrollToForm)
+  form.on('prevPage', scrollToForm)
```

Comment:
I'm unconvinced that going back a page should scroll you to the top. I think going back to where you were before seems what I'd expect.